hugr-llvm/src/extension/prelude/array.rs

```diff
@@ -779,4 +938,191 @@ mod test {
         exec_ctx.add_extensions(|cge| cge.add_default_prelude_extensions().add_int_extensions());
         assert_eq!(expected, exec_ctx.exec_hugr_u64(hugr, "main"));
     }
+
+    #[rstest]
+    #[case(5, 42, 0)]
+    #[case(5, 42, 1)]
+    #[case(5, 42, 2)]
+    #[case(5, 42, 3)]
+    #[case(5, 42, 4)]
+    fn exec_repeat(
```

Comment:
This isn't a very satisfying test -- it'd be nicer to have something where the function inspects the inputs and puts a different value in each location

Reply:
That's not possible with repeat, all values in the array must have the same value

Reply:
Ah I see, I was thinking repeat was haskell's `Data.List.iterate`